mysql-compatibility.md

```diff
@@ -104,34 +61,23 @@ TiDB 支持常用的 MySQL 内建函数，但是不是所有的函数都已经
 
 ### DDL
 
-在 TiDB 中，运行的 DDL 操作不会影响对表的读取或写入。但是，目前 DDL 变更有如下一些限制：
-
-+ Add Index
-    - 不支持同时创建多个索引
-    - 其他类型的 Index Type (HASH/BTREE/RTREE) 只有语法支持，功能不支持
-+ Add Column
-    - 不支持将新创建的列设为主键或唯一索引，也不支持将此列设成 AUTO_INCREMENT 属性
-+ Drop Column: 不支持删除主键列或索引列
-+ Change/Modify Column
-    - 不支持有损变更，比如从 `BIGINT` 变为 `INTEGER`，或者从 `VARCHAR(255)` 变为 `VARCHAR(10)`
-    - 不支持修改 `DECIMAL` 类型的精度
-    - 不支持更改 `UNSIGNED` 属性
-    - 只支持将 `CHARACTER SET` 属性从 `utf8` 更改为 `utf8mb4`
-+ `LOCK [=] {DEFAULT|NONE|SHARED|EXCLUSIVE}`: TiDB 支持的语法，但是在 TiDB 中不会生效。所有支持的 DDL 变更都不会锁表。
-+ `ALGORITHM [=] {DEFAULT|INSTANT|INPLACE|COPY}`: TiDB 完全支持 `ALGORITHM=INSTANT` 和 `ALGORITHM=INPLACE` 语法，但运行过程与 MySQL 有所不同，因为 MySQL 中的一些 `INPLACE` 操作实际上是 TiDB 中的 `INSTANT` 操作。`ALGORITHM=COPY` 语法在 TiDB 中不会生效，会返回警告信息。
-+ 单个 `ALTER TABLE` 语句中无法完成多个操作。例如，不能用一个语句来添加多个列或多个索引。
-+ Table Option 不支持以下语法
-    - `WITH/WITHOUT VALIDATION`
-    - `SECONDARY_LOAD/SECONDARY_UNLOAD`
-    - `CHECK/DROP CHECK`
-    - `STATS_AUTO_RECALC/STATS_SAMPLE_PAGES`
-    - `SECONDARY_ENGINE`
-    - `ENCRYPTION`
-+ Table Partition 不支持以下语法
-    - `PARTITION BY LIST`
-    - `PARTITION BY KEY`
-    - `SUBPARTITION`
-    - `{CHECK|EXCHANGE|TRUNCATE|OPTIMIZE|REPAIR|IMPORT|DISCARD|REBUILD|REORGANIZE} PARTITION`
+在 TiDB 中，运行的 DDL 操作不会影响对表的读取或写入。但是，目前 DDL 以下变更和 MySQL 的 DDL 变更有所区别：
+
+| 语句类型 | 描述 | 错误信息示例 |
+| ------ | ---- | ------ | ------ |
+| Alter Table | 不支持在单个 alter 语句中指定多个 alter 选项 | Unsupported multi schema change |
+| Table Option | 表选项除了 `AUTO_INCREMENT`、`CHARACTER SET`、`COLLATE`、`COMMENT` 以外，其他所有的选项都会被忽略 | N/A |
```

Comment:
auto_id_cache、auto_random_base

Reply:
MySQL doesn't support them, I think they can be put to `auto-increment.md` and `auto-random.md`.

Reply:
ok